segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessorTest.java

```diff
@@ -598,6 +598,21 @@ public void testSegmentAttribute() throws Exception {
         order.verify(connection).send(new WireCommands.SegmentAttribute(9, WireCommands.NULL_ATTRIBUTE_VALUE));
     }
 
+    @Test(timeout = 200000)
```

Comment:
Why is this timeout so long?

Reply:
Will have to calibrate, but flushing takes time.